docs/product/relay/projects.mdx

```diff
@@ -108,3 +108,17 @@ An empty list rejects all origins. Use the default `["*"]` to allow all origins.
   }
 }
 ```
+
+`config.features`
+
+Enable product features that are not yet enabled by default.
+
+```json
+{
+  "config": {
+    "features": ["organizations:session-replay"]
+  }
+}
+```
+
+A full list of features can be found [here](https://github.com/getsentry/relay/blob/3dff617a1a5ff01f6617de82807bc50c7d695a2c/relay-dynamic-config/src/feature.rs).
```

Comment:
We probably should link to `master` here, so we don't have to continously update the link.

Best would be though if we could use the [Rust docs](https://getsentry.github.io/relay/relay_dynamic_config/enum.Feature.html) but it does not show the feature string.

Reply:
Good point, I wanted to link to master, don't know why I copied to SHA link in the end.

> Best would be though if we could use the [Rust docs](https://getsentry.github.io/relay/relay_dynamic_config/enum.Feature.html) but it does not show the feature string.

That's where I looked initially. We could consider copying the string repr into the doc comment for every variant.

Reply:
I think we should and `#[doc(hidden)]` the deprecated ones.

Reply:
https://github.com/getsentry/relay/pull/3236